### ISSUE TITLE
[FeatureRequest] CloseChannelSync gRPC function

### ISSUE THREAD

**Issue by benthecarman** [lightningnetwork/lnd#5171]
It would be nice to have a `CloseChannelSync` function that closes a channel without having to manage a stream. The return could be the transaction or txId of the closing tx.

**Comment by Roasbeef** [comment]
You don't really _need_ to manage the stream. Just reading the very first value (blocking until that's present) works. Otherwise you can hit it over the REST interface in a similar manner.

**Comment by benthecarman** [comment]
> You don't really need to manage the stream. Just reading the very first value (blocking until that's present) works. Otherwise you can hit it over the REST interface in a similar manner.

Yeah I guess it'd just be nice to have a function that did that for me. I am working with the gRPC in scala with akka and to turn the stream into what I want it requires me to add a bunch of extra things to the call instead of just being able to call `.invoke()`
